QuizApp/QuizApp/Classes/Presentation/MainMenu/MainMenuViewController.swift

```diff
@@ -8,4 +8,42 @@
 
 import UIKit
 
-class MainMenuViewController: UIViewController {}
+final class MainMenuViewController: UIViewController {
+    @IBOutlet private var playMenuButtom: MenuButton!
+    @IBOutlet private var rankingsMenuButton: MenuButton!
+    @IBOutlet private var settingsMenuButton: MenuButton!
+    
+    var presenter: MainMenuPresenter?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureButtons()
+    }
+    
+    private func configureButtons() {
+        playMenuButtom.configure(with: "Play".localized, and: .play)
+        rankingsMenuButton.configure(with: "Rankings".localized, and: .rankings)
+        settingsMenuButton.configure(with: "Settings".localized, and: .settings)
+        
+        let presenter = self.presenter
+        playMenuButtom.onTapAction = { presenter?.onPlayPressed() }
```

Comment:
```suggestion
        playMenuButtom.onTapAction = presenter?.onPlayPressed
```
так компилятор пропустит?